src/cryptography/hazmat/primitives/asymmetric/utils.py

```diff
@@ -28,6 +29,9 @@ def decode_rfc6979_signature(signature):
         raise ValueError(
             "The signature contains bytes after the end of the ASN.1 sequence."
         )
+    if data == eoo.endOfOctets:
```

Comment:
Can you add a comment about the fact that this occurs due to a PyASN1 bug, and can be removed in a hypothetical future where eoo is handled correctly?